bower.json

```diff
@@ -6,6 +6,7 @@
   },
   "devDependencies": {
     "requirejs": "~2.1.9",
+    "immutable": "~3.6.2",
```

Comment:
@cutandpastey For non-cached builds I felt it was necessary to have it at the right place (it didn't work for me until I added it anyway)